src/spring-cloud/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+2.1.1
+-----
+* Remove preview parameter and fix warning message.
```

Comment:
Can you add the parameter name as well? And it's only removed in `update`, right?

Reply:
No, I just removed "--enable-java-agent" from "update" command, I didn't add new parameter.

Reply:
I mean make the notes clear like `Remove preview parameter --enable-java-agent from az spring-cloud update`

Reply:
ok,I see. fixed it.